lib/sensors/include/URM04Sensor.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "mbed.h"
+
+namespace URM04Sensor {
+
+class URM04Sensor {
+ protected:
+  // constants
+  static constexpr int BAUD_RATE     = 19200;
+  static constexpr int START_ADDRESS = 0x11;
+  static constexpr int LOW           = 0;
+  static constexpr int HIGH          = 1;
+  static constexpr float MAX_FLOAT   = 1e+37;
```

Comment:
Use `std::numeric_limits<float>::max()` instead of a hardcoded value. https://www.cplusplus.com/reference/limits/numeric_limits/